hooks/pre_commit/42-evr_extension_permissions.rb

```diff
@@ -0,0 +1,31 @@
+# Managed databases will be handled automatically.
+return if local_postgresql?
+
+database = param_value('foreman', 'db_database') || 'foreman'
+config = load_db_config(database)
```

Comment:
```suggestion
config = load_db_config('foreman')
```

While you can deploy the Foreman DB with a different DB name (no idea why you would, but you can), `load_db_config` only accepts `foreman` (and `candlepin` and `pulpcore`) keys.